Source/Redemption/Public/MainMenuWidget.h

```diff
@@ -23,6 +23,15 @@ class REDEMPTION_API UMainMenuWidget : public UUserWidget
 	UPROPERTY(BlueprintReadOnly, meta = (BindWidget))
 	UButton* QuitButton;
 
+	UPROPERTY(BlueprintReadOnly, meta = (BindWidget))
+	UButton* OptionsButton;
+
+	
+	TSubclassOf<UUserWidget>OptionsMenuClass;
```

Comment:
this variable is not used, you can remove it, and there are some empty lines that can be removed